python/schedule.hpp

```diff
@@ -34,6 +35,8 @@ struct regular_schedule_shim {
     opt_time_type get_tstop()  const;
 
     arb::schedule schedule() const;
+
+    std::vector<arb::time_type> events(pybind11::object t0, pybind11::object t1);
```

Comment:
Why not just use `arb::time_type` instead of `pybind11::object` here, and let pybind11 do the conversion?

Reply:
Since we allowed `None` in the regular schedule as inputs, so I would assume as a user that `None` would also be possible to get the list of events.

Reply:
This interface explicitly requests a start time and end time for the interval. What would `None` mean in this context?

Reply:
Same as for the `regular_schedule` - no start time, no end time. Thus, no events, or all events. Happy to change it to `arb::time_type` if these are no user cases.

Reply:
I think that all the schedules should have the same interface of `(t0, t1)` being valid `time_type`s.

Reply:
Ok.

Reply:
Done.